extensions/3rdparty/tabber/tabber.php

```diff
@@ -57,18 +57,18 @@ function renderTabber( $paramstring, $params, $parser ){
  * @param Parser $parser
  * @return string
  */
-function buildTab($tab, $parser){
-	if( trim($tab) == '' ) return '';
+function buildTab( $tab, $parser ) {
+	if ( trim( $tab ) == '' ) return '';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)